src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -327,25 +327,16 @@ export const makeDatasetTableHeader = (datasets: DatasetTerm[], selected: number
       label: 'Access Type',
       sortable: true,
       cellStyle: makeHeaderStyle(cellWidths.accessType),
-      cellDataFn: (dataset: DatasetTerm) => {
-        let accessType;
-        if (dataset.accessManagement === 'external') {
-          accessType = dataset.url ?
-            <Link href={dataset.url}>External to DUOS</Link> : 'External to DUOS';
-        } else if (dataset.accessManagement === 'open') {
-          accessType = dataset.url ? <Link href={dataset.url}>Open Access</Link> : 'Open Access';
-        } else {
-          accessType = dataset.dac?.dacEmail ? <Link
-            href={'mailto:' + dataset.dac.dacEmail}>{dataset.dac?.dacName}</Link> : dataset.dac?.dacName;
-        }
-        return {
-          data: accessType,
-          value: dataset.accessManagement,
-          id: `${dataset.datasetId}-participant-count`,
-          style: makeRowStyle(cellWidths.accessType),
-          label: `Access Type for dataset ${dataset.datasetId}: ${dataset.accessManagement}`
-        };
-      }
+      cellDataFn: (dataset: DatasetTerm) => ({
+        data: dataset.accessManagement === 'external' ?
+          'External to DUOS' :
+          dataset.accessManagement === 'open' ?
+            'Open Access' : dataset.dac?.dacName,
+        value: dataset.accessManagement,
+        id: `${dataset.datasetId}-participant-count`,
```

Comment:
Do you know why the id is named "participant-count" ?

Reply:
Probably a copy-paste from a previous column. We should probably give this a unique name.